doc/scripts/filter-doc-log.sh

```diff
@@ -11,15 +11,25 @@ CONFIG_DIR=${ZEPHYR_BASE}/.known-issues/doc
 
 LOG_FILE=$1
 
+red='\E[31m'
+
 if [ -z "${LOG_FILE}" ]; then
         echo "Error in $0: missing input parameter <logfile>"
         exit 1
 fi
 
-if [ -e "${LOG_FILE}" ]; then
-   if [ -s "${LOG_FILE}" ]; then
-      $KI_SCRIPT --config-dir ${CONFIG_DIR} ${LOG_FILE}
+if [ -s "${LOG_FILE}" ]; then
+   $KI_SCRIPT --config-dir ${CONFIG_DIR} ${LOG_FILE} > doc.warnings 2>&1
+   if [ -s doc.warnings ]; then
+	   echo
+	   echo -e "${red}New errors/warnings found, please fix them:"
+	   echo -e "=============================================="
+	   tput sgr0
+	   echo
+	   cat doc.warnings
+	   echo
```

Comment:
How about adding an else after this indicating there were no "new" errors/warnings:

Reply:
yes, good idea